lib/repo/subversion_repository.rb

```diff
@@ -46,8 +46,8 @@ def initialize(connect_string)
       rescue NotImplementedError; end
       @repos_path = connect_string
       @closed = false
-      @repos_auth_file = Repository.conf[:REPOSITORY_PERMISSION_FILE] || File.dirname(connect_string) + "/svn_authz"
-      @repos_admin = Repository.conf[:IS_REPOSITORY_ADMIN]
+      @repos_auth_file =  MarkusConfigurator.markus_config_repository_permission_file || File.dirname(connect_string) + "/svn_authz"
```

Comment:
Line is too long. [132/80]<br>Unnecessary spacing detected.<br>Operator `=` should be surrounded by a single space.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.